src/platform/api.py

```diff
@@ -132,7 +132,7 @@ def resolve_user(user_id: str):
 def whoami() -> dict:
     """get details about currently signed in user"""
 
-    return _make_request("/users/me")
+    return _make_request("/users/me", verb="PATCH")
```

Comment:
This seems sketchy

Reply:
this will go away when we use the auto generated client